Sprint-3/alarmclock/alarmclock.js

```diff
@@ -1,4 +1,143 @@
-function setAlarm() {}
+const userInput = document.getElementById("alarmSet").value;
+
+//split function
+const splitTime = (userInput) => {
+  return userInput.split(":");
+};
+
+//parse time to a format we can work with
+const splitAndParseTime = (userInput) => {
```

Comment:
I'm not sure that the requirements here were to include an hours column - it looks like it's just minutes and seconds, in an MM:SS format.